src/snapshot_middleware/txt.rs

```diff
@@ -14,10 +13,8 @@ pub fn snapshot_txt(
     path: &Path,
     name: &str,
 ) -> anyhow::Result<Option<InstanceSnapshot>> {
-    let contents = vfs.read(path)?;
-    let contents_str = str::from_utf8(&contents)
-        .with_context(|| format!("File was not valid UTF-8: {}", path.display()))?
-        .to_owned();
+    let contents = vfs.read_to_string_lf_normalized(path)?;
+    let contents_str = contents.as_str();
```

Comment:
I'm inclined to say that normalizing line endings for StringValues is a bad call since Roblox themselves don't, so we'd be inventing behavior. The reason we're bothering with scripts is that Roblox does it and we want to match their behavior.

Reply:
Right - it's just that letting CRLF line endings through for text files could bite teams that don't all work on the same OS. The only thing that could possibly go "wrong" is annoying diffs, and they could (and definitely should) configure their VCS to normalize line endings. 

I'm open to leaving it out of this PR, but we should stick a pin into it in case anyone runs into this issue again.